src/Pages/Applications/DevtronApps/Details/AppConfigurations/Navigation/EnvironmentOverrideRouter.tsx

```diff
@@ -279,29 +226,23 @@ const ValueContainer = (props: ValueContainerProps) => {
     )
 }
 
-const EnvironmentOverrideRouter = ({
-    isJobView,
-    workflowsRes,
-    getWorkflows,
-    allEnvs,
-    reloadEnvironments,
-}: EnvironmentOverrideRouterProps) => {
+const EnvironmentOverrideRouter = () => {
     const { pathname } = useLocation()
     const { appId } = useParams<{ appId: string }>()
     const previousPathName = usePrevious(pathname)
     const [environmentOptions, setEnvironmentOptions] = useState([])
-    const [jobEnvs, setJobEnvs] = useState([])
     const [addEnvironment, setEnvironmentView] = useState(true)
     const [ciPipelines, setCIPipelines] = useState([])
     const [isEnvLoading, setIsEnvLoading] = useState(false)
 
+    const { isJobView, getWorkflows, reloadEnvironments, environments } = useAppConfigurationContext()
+
     const getJobOtherEnvironment = async () => {
         setIsEnvLoading(true)
         try {
-            const [{ result: envListMinRes }, { result: ciConfigRes }, { result: jobEnvRes }] = await Promise.all([
+            const [{ result: envListMinRes }, { result: ciConfigRes }] = await Promise.all([
                 getEnvironmentListMinPublic(),
```

Comment:
Plesae check if this is required else remove

Reply:
it is required, this is used in case of job to fetch the environments of job not same as app environments